io.embrace.sdk/Scripts/EmbraceStartupArgs.cs

```diff
@@ -0,0 +1,20 @@
+namespace EmbraceSDK.Internal
+{
+    public struct EmbraceStartupArgs
+    {
+        public string AppId;
+        public string AppGroupId;
+        public string BaseUrl;
+        public string DevBaseUrl;
+        public string ConfigBaseUrl;
+        
+        public EmbraceStartupArgs(string appId, string appGroupId, string baseUrl, string devBaseUrl, string configBaseUrl)
```

Comment:
Most of these should be optional/have default values IMO

Reply:
Noted. Will do.

Reply:
I plan to include defaults for all of them *except* AppID if that's OK?
The intention there is to force the user to input their target AppID as, for now, Unity has yet to expose functionality for initialization without an AppID